src/Compilers/Core/Portable/DiagnosticAnalyzer/CompilationWithAnalyzers.cs

```diff
@@ -775,13 +778,13 @@ private async Task<(ImmutableArray<CompilationEvent> events, bool hasSymbolStart
             }
         }
 
-        private async Task<ImmutableArray<CompilationEvent>> GenerateCompilationEventsAndPopulateEventsCacheAsync(AnalysisScope analysisScope, AnalyzerDriver driver, CancellationToken cancellationToken)
+        private async Task GenerateCompilationEventsAndPopulateEventsCacheAsync(AnalysisScope analysisScope, AnalyzerDriver driver, CancellationToken cancellationToken)
         {
 #if SIMULATED_EVENT_QUEUE
             await _analysisState.GenerateSimulatedCompilationEventsAsync(analysisScope, _compilation, _compilationData.GetOrCreateCachedSemanticModel, driver, cancellationToken).ConfigureAwait(false);
```

Comment:
>await _analysisState.GenerateSimulatedCompilationEventsAsync [](start = 12, length = 60)

Seems like this line would not compile in previous method signature. Do we ever compiler this line?

Reply:
Yes, that was a prior implementation when we were trying out. Let me file a separate work item to remove all code under `SIMULATED_EVENT_QUEUE`

Reply:
Filed #37434